examples/nlp/text_classification_tf_keras/adaptive.yaml

```diff
@@ -0,0 +1,21 @@
+description: text_classification_tf
```

Comment:
lets make this `text_classification_tf_adaptive`

Reply:
done